keep/api/bl/ai_suggestion_bl.py

```diff
@@ -324,7 +324,7 @@ async def commit_incidents(
             try:
                 # Create the incident
                 incident_dto = IncidentDto.parse_obj(incident_with_feedback["incident"])
-                created_incident = incident_bl.create_incident(incident_dto)
+                created_incident = incident_bl.create_incident(incident_dto, generated_from_ai=True)
```

Comment:
@35C4n0r is this intended?

Reply:
Yes, generated_from_ai indicates if the Incident is created by the Keep's AI